net-libs/cppzmq/files/cppzmq-use-system-catch2.patch

```diff
@@ -0,0 +1,29 @@
+--- a/tests/CMakeLists.txt
++++ b/tests/CMakeLists.txt
+@@ -9,9 +9,12 @@
+ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR})
+ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/${CMAKE_INSTALL_BINDIR})
+ 
++find_package(Catch2 REQUIRED)
+ include(CTest)
+-include(cmake/catch.cmake)
+-include(${CATCH_MODULE_PATH}/Catch.cmake)
++#include(cmake/catch.cmake)
```

Comment:
Please don't leave stuff commented out in patches. This just makes them longer, and `-` context is sufficient to know what you've removed.

Reply:
done